.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
```

Comment:
Could you just comment that it is only for Sonar compatibility? Assuming that we deliver a java8-compatible artifact

Reply:
+1 (we still build for Java 8 as a target)